### PR TITLE
pkg/config: more test

### PR DESCRIPTION
**Pull request by chai2010**

### CODE REVIEW

File: pkg/config/app/app_config_test.go [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+func TestConfig_default(t *testing.T) {
+	cfg, err := LoadConfig()
+	tAssert(t, err == nil, err)
+
+	var pb_const pb.Const
+	tAssert(t, cfg.App.Host == pb_const.GetAppHost())
+	tAssert(t, cfg.App.Port == int(pb_const.GetAppPort()))
+
+	tCheckDefaultGlog(t)
+	tCheckDefultDB(t)
```
Comment (chilianyi) [inline pkg/config/app/app_config_test.go]: default?

Comment (chai2010) [reply pkg/config/app/app_config_test.go]: it means default db config

Comment (chilianyi) [reply pkg/config/app/app_config_test.go]: Sorry, I mean the spell

Comment (chai2010) [reply pkg/config/app/app_config_test.go]: it is a mistake, i will fix it. thx

**Pull request merged**